djedi/plugins/form.py

```diff
@@ -0,0 +1,56 @@
+import json
+import cio
+from cio.plugins.base import BasePlugin
+from django import forms
+
+
+class BaseEditorForm(forms.Form):
+    def __init__(self, *a, **kw):
```

Comment:
IMO we should try and stick to the established `*args` and `**kwargs` naming scheme for these arguments.

Reply:
Agreed, unintentionally committed :)